### PR TITLE
fix a bug of device update

### PR DESCRIPTION
**Pull request by wuqihui0317**
When I update a device's propertyVisitor, cm isn't updated.

### CODE REVIEW

File: cloud/pkg/devicecontroller/controller/downstream.go [hunk 1]
```diff
@@ -489,6 +489,11 @@ func isDeviceDataUpdated(oldData *v1alpha2.DeviceData, newData *v1alpha2.DeviceD
 	return !reflect.DeepEqual(oldData, newData)
 }
 
+// isDeviceDataUpdated checks if DeviceData is updated
```
Comment (chendave) [inline cloud/pkg/devicecontroller/controller/downstream.go:492]: isDevicePropertyVisitorsUpdated checks if PropertyVisitors has been updated

Comment (wuqihui0317) [reply cloud/pkg/devicecontroller/controller/downstream.go]: Thanks, I modify this.

**Comment by GsssC** [comment]
need squash your commits to one.
:)

**Comment by wuqihui0317** [comment]
> need squash your commits to one.
> :)

ok

**Comment by fisherxu** [comment]
/approve
Thanks for the fixing :)

**Comment by fisherxu** [comment]
/approve
Thanks for the fixing :)

**Pull request merged**